File: Polimorfismo - Batalha Final/Java/Chefe.java
class Chefe extends Personagem{

    public Chefe(String nome){
        super(nome, 200, 20);
    }
    
    @Override
    public void ficha(){
        System.out.println("%n[CHEFE]:%n");
        super.ficha();
        
    }
    
    @Override
    public String habilidade(){
        return nome + "usa Ataque devastador";
    }
    
}

File: Polimorfismo - Batalha Final/Python/Chefe.py
from Personagem import Personagem

class Chefe(Personagem):

    def __init__(self, nome):
        super().__init__(nome, 200, 20)

    def ficha(self):
        print("\n[CHEFE]:\n")
        super().ficha()

    def habilidade(self):
        return self.nome + "usa Ataque devastador"